docs/getting-started.md

```diff
@@ -21,7 +21,7 @@ _To learn more about the different installation methods, refer to the [Installat
 
 ### Local Installation
 
-If you're running Linux, macOS, or Windows and have [Python](https://www.python.org/) 3.7, 3.8 or 3.9 installed,
+If you're running Linux, macOS, or Windows and have a supported version of [Python](https://www.python.org/) installed,
```

Comment:
I think we should be explicit in the versions we support. I'm not sure we mention supported versions elsewhere and I don't expect people to go look at the repo to figure it out.

Reply:
I prefer writing docs that aren't guaranteed to fall out of date 1-2 times per year as we add/remove supported Python versions, but I'll compromise here since having users need to check `pyproject.toml` or https://pypi.org/project/meltano/ for the version info while they're reading the getting-started guide is not ideal. In generally we should prefer not duplicating this information in many places if we can help it.